notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    tbl=tbl[~tbl['qmin'].mask] # Is this gross? 
</pre><p>Remove the comment if no one has objections :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>